src/components/Modal.js

```diff
@@ -73,7 +82,7 @@ class Modal extends React.Component {
   _renderTooltipIconAndLabel () {
     if (this.props.tooltip) {
       return (
-        <div>
+        <div className='mx-modal-tooltip-label' style={styles.tooltipLabel}>
```

Comment:
Somewhat related...we don't need the else statement below. If there is no tooltip then just don't do anything in this method.